File: Data Structures & Algorithms/remove-duplicates-from-sorted-array/submission-4.py
class Solution:
    def removeDuplicates(self, nums: List[int]) -> int:
        if not nums:
            return 0
        k=1
        L = 0
        for R in range(1,len(nums)):
            if nums[L]!=nums[R]:
                L+=1
                nums[L]=nums[R]
                k+=1
            else:
                pass

        return k

File: Data Structures & Algorithms/remove-element/submission-4.java
class Solution {
    public int removeElement(int[] nums, int val) {
        int currIndex = 0;
        int k = 0;

        for (int i = 0; i<nums.length; i++) {
            if (nums[i]!=val) {
                nums[currIndex]=nums[i];
                currIndex++;
                k++;
            }
        }

        return k;
    }
}